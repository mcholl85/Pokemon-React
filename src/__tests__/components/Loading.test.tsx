import { render } from '@testing-library/react'
import Loading from '../../components/Loading'

describe('Loading', () => {
  test('Matches DOM snapshot', () => {
    const { asFragment } = render(<Loading />)

    expect(asFragment()).toMatchSnapshot()
  })
})
