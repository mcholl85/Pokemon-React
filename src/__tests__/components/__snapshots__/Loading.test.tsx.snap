// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Matches DOM snapshot 1`] = `
<DocumentFragment>
  <div>
    Loading...
  </div>
</DocumentFragment>
`;
